docs/_jul_milestone_.md

```diff
@@ -0,0 +1,60 @@
+# Monthly Milestone - July 2020
+
+## Overview
+
+Focus on:
+
+- documentation improvements
+- reduce backlog of PRs waiting to be reviewed
+
+## Goals
+
+- [ ] Docs improvements #5113
+
+  - [ ] Hide @loopback/openapi-v3 from docs, examples and CLI templates #5692
+  - [ ] Docs: how to configure a custom table/collection name different from
+        model class name #2197
+  - [ ] Improve and refactor the authentication documentations #5294
+
+- [ ] Feature parity
+
+  - [ ] Continue with feat(repository): hasManyThrough #4438
+
+- [ ] Fix GVT issues
+
+  - [ ] [GSSC-GVT]English String when input command apic lb4 app -h #5654
+  - [ ] [GSSC-GVT]English String when input command apic lb4 repository #5656
+  - [ ] [GSSC-GVT]English String when input command apic lb4 model #5657
+  - [ ] [GSSC-GVT]English String when input command APIC apic lb4 interceptor
+        #5658
+
+- [ ] Bugs
+
+  - [ ] Regression: MongoError: unknown operator: \$\$eq
+        strongloop/loopback-connector-mongodb#532
+
+- [ ] PRs to review
+  - [ ] Add "signup" / create user in authentication-jwt module #5602
+  - [ ] 5187 jwt refresh token Feature #5589
+  - [ ] feat(pooling): add an extension to provide pooling service #5681
+  - [ ] [RFC] Define sequence actions as middleware #5366
+  - [ ] [RFC WIP] feat(typegoose): add a component for typegoose/mongoose
+        integration #5152
+
+## Stretch Goals
+
+- [ ] Docs Improvement
+
+  - [ ] couchDB connector docs #5267
+  - [ ] [Docs] Add "project layout reference" page #4847
+  - [ ] Add missing LoopBack 4 Types docs #5063
+
+- [ ] Feature parity
+
+  - [ ] Execute raw NoSQL queries (e.g. MongoDB) #3342
+
+- [ ] Chores/Tooling improvements
+
+  - [ ] update dependency mem-fs-editor to v7 #5624
+  - [ ] [api-extractor] Incorrect handling of interfaces enhancing a class with
+        the same name microsoft/rushstack#1921
```

Comment:
We found a (workaround) solution that does not involve interfaces enhancing a class. Let's remove this task from the milestone.